src/plugins/presentation_panel/public/panel_component/_presentation_panel.scss

```diff
@@ -184,3 +169,57 @@
   padding-left: $euiSizeS;
   z-index: $euiZLevel1;
 }
+
+.embPanel__hoverActionsAnchor {
+  position: relative;
+  height: 100%;
+
+  .embPanel__hoverActionsWrapper {
+    height: $euiSizeXL;
+    position: absolute;
+    top: 0;
+    display: flex;
+    justify-content: space-between;
+    padding: 0 $euiSize;
+    flex-wrap: nowrap;
+    min-width: 100%;
+    z-index: -1;
+    pointer-events: none; // Prevent hover actions wrapper from blocking interactions with other panels
+  }
+
+  .embPanel__hoverActions {
+    opacity: 0;
+    padding: $euiSizeXS;
+    display: flex;
+    flex-wrap: nowrap;
+    border: $euiBorderThin;
+
+    background-color: $euiColorEmptyShade;
+    height: $euiSizeXL;
+
+    pointer-events: all; // Re-enable pointer-events for hover actions
+  }
+
+  .embPanel--dragHandle {
+    cursor: move;
+
+    img {
+      pointer-events: all !important;
+    }
+  }
+
+  .embPanel__descriptionTooltipAnchor {
+    padding: $euiSizeXS;
+  }
+
+  &:hover .embPanel__hoverActionsWrapper,
+  &:focus-within .embPanel__hoverActionsWrapper,
+  .embPanel__hoverActionsWrapper--openContextMenu {
+    z-index: $euiZLevel9;
+    top: -$euiSizeXL;
+
+    .embPanel__hoverActions {
+      opacity: 1;
+    }
+  }
+}
```

Comment:
Could we write these styles using emotion especially that eui tokens are used here? There's an upcoming theming initiative that would effectively phase out the usage of styled components, and possibly the usage scss files.

Reply:
Addressed in [6a7982c](https://github.com/elastic/kibana/pull/182535/commits/6a7982c4fb5a531a25094dae9b02edac26610a46).

I did leave some styles in the scss file though because I couldn't figure out how to get the complex selector to work with Emotion to handle the styles on hover. See https://github.com/elastic/kibana/pull/182535/files#diff-d4f343c3886df5fe9a99f56aeaf035c5e7d8723062fc689f6b889398b3c41d6eR172-R198

Reply:
Honestly, I wonder if we can just stick to class-based styling right now and create a follow up issue to turn **everything** into Emotion as a follow up?

I personally find it confusing to have some styles in a SASS file while others are split off into Emotion when they are both targeting the same component - so unless we can **fully** convert these styles to Emotion (which is definitely possible, it would just take some work).... I'm not sure if I'm a fan of the current split implementation? And I don't think we should block this PR based on this.

Reply:
> Honestly, I wonder if we can just stick to class-based styling right now and create a follow up issue to turn **everything** into Emotion as a follow up?
> 
> I personally find it confusing to have some styles in a SASS file while others are split off into Emotion when they are both targeting the same component - so unless we can **fully** convert these styles to Emotion (which is definitely possible, it would just take some work).... I'm not sure if I'm a fan of the current split implementation? And I don't think we should block this PR based on this.

I do agree a split to target the same component is somewhat convoluted, my comment was made under the assumption that all the styles would be written to emotion and there's no reason to block this PR, just that the conversion to emotion is an eventuality.